templates/example.gotxt

```diff
@@ -2,7 +2,11 @@
 
 {{- template "doc" .Doc -}}
 
-{{- codeBlock "go" .Code -}}
+{{- codeBlock "go" .Code }}
+
+Output:
+
+{{ codeBlock "output" .Output -}}
```

Comment:
Not all examples have output. This section should probably be omitted if the output is empty. Something like:

```suggestion
{{- if .HasOutput -}}
	{{- header 4 "Output" -}}

	{{- codeBlock "" .Output -}}
{{- end -}}
```

Reply:
Implemented in latest commit